br/pkg/gluetidb/glue_test.go

```diff
@@ -20,21 +20,37 @@ import (
 
 	"github.com/pingcap/tidb/br/pkg/glue"
 	"github.com/pingcap/tidb/pkg/parser/model"
+	"github.com/pingcap/tidb/pkg/session"
 	"github.com/pingcap/tidb/pkg/testkit"
 	"github.com/pingcap/tidb/pkg/types"
 	"github.com/stretchr/testify/require"
 )
 
 func TestTheSessionIsoation(t *testing.T) {
 	req := require.New(t)
-	store := testkit.CreateMockStore(t)
+	store, dom := session.CreateStoreAndBootstrap(t)
 	ctx := context.Background()
 
-	g := Glue{}
-	session, err := g.CreateSession(store)
+	// we want to test glue start domain explicitly, so close it first.
+	dom.Close()
```

Comment:
is there case covering the case where dom is opened already?

Reply:
check glue.GetDomain, it will get session twice, domain has started when create the second one